README.md

```diff
@@ -148,6 +150,50 @@ To mask obsoleted tests and prevent them from running by `brownie test` even
 when residing in archive directory, please consider to rename them:
 `test_` → `xtest_`.
 
+## Common tests
+
+There are two groups of common tests in `tests` directory: regression
+(`tests/test_regression*.py`) and snapshot (`tests/test_snapshot*.py`).
```

Comment:
I propose to move the common tests to subdirs, i.e. 
* `tests/common/regression`
* `tests/common/snapshot`